Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:20.04
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    python-is-python3 \
+    python3 \
+    python3-pip \
+    && rm -rf /var/lib/apt/lists/*
+COPY . /ml-compiler-opt
+RUN python3 -m pip install -r /ml-compiler-opt/requirements.txt
+VOLUME /external
```

Comment:
This is just a nit, but there should be a newline at end of file.

Reply:
Fixed.